tests/utils/deleteNearbyEntities.js

```diff
@@ -0,0 +1,6 @@
+// Remove all entities near the avatar
+const TEST_RADIUS = 1000;
+var entitiesToDelete = Entities.findEntities(MyAvatar.position, TEST_RADIUS);
```

Comment:
I wonder if it wouldn't be better to just delete the entities in view, which I think you can do with `Entities.findEntitiesInFrustum(Camera.frustum)`.

Reply:
That would be risky.  Some tests rotate the viewpoint during the test.  One
that does that for sure is the camera test that tests camera rotation…

Remember that this only done during automatic testing.  There shouldn't be
any entities at all.  But, especially during testing, it has happened.
Adam has seen strange results sometimes, and this seems to be the problem.

Reply:
ah ok good point